src/App.js

```diff
@@ -1,27 +1,38 @@
-import logo from './logo.svg';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from 'react-router-dom';
+
 import './App.css';
 
+import { Provider } from 'react-redux';
+import store from './redux/store';
+import HomePage from './pages/HomePage';
+import Regionpage from './pages/RegionPage';
+import Worldpage from './pages/WorldPage';
+
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div data-testid="mainApp">
+      <Provider store={store}>
```

Comment:
- [ ] OPTIONAL
It's a good idea to place the `<Provider>`at the top level, for instance, inside `index.js`. Kindly refer to this [documentation](https://react-redux.js.org/api/provider) for further details.